StitchMDBKitExample/StitchMDBKitExample/ToDoListViewController.swift

```diff
@@ -0,0 +1,132 @@
+import MongoMobile
+import StitchCore_iOS
+import StitchMDBKit
+import Toast_Swift
+import UIKit
+
+/// Name of ToDoList db
+private let toDoListDatabaseName = "todo"
+/// Name of ToDoList collection
+private let toDoListCollectionName = "items"
+/// Cell reuse id (cells that scroll out of view can be reused)
+private let cellReuseIdentifier = "ToDoItemTableViewCell"
+
+/// View Controller for the ToDo List
+class ToDoListViewController: UITableViewController {
+    private var toDoListCollection: MongoCollection!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        do {
+            // try to initialize the default stitch client
+            let client =
+                try Stitch.initializeDefaultAppClient(
```

Comment:
[nit] not that it matters all that much for this example, but if this eventually going to be a public example, this should probably be called in `application(didFinishLaunching...` with the mongomobile init code and here should be `getDefaultAppClient` since that will probably be the best practice. If this VC gets loaded multiple times (if for example someone modified the example so that this VC isn't the top-level VC), this would crash the app.

Reply:
I was going by what we do in other examples. If we go with that approach, we should do the equivalent on Android as well.